packages/design-system/src/components/Icons/MinusCircleIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { t } from '../i18n';
+import SvgIcon, { IconCommonProps } from './SvgIcon';
+import classNames from 'classnames';
+
+export interface MinusCircleIconProps extends IconCommonProps {
+  isFilled?: boolean;
```

Comment:
Is this prop actually used? It looks like you're managing the fill state with CSS